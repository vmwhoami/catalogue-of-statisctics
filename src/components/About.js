import css from '../styles/about.module.css';
import { VmLogo } from './Svgs';

const About = () => (
  <div className={css.container}>
    <VmLogo style={css.logo} />
    <div className={css.content}>
      <h2>This is a websited that displays the All the crypto that a worth more than a dollar</h2>
      <p>
        Lorem ipsum dolor sit amet consectetur adipisicing elit.
        Nesciunt dolor eius rem voluptates aliquid, itaque labore quae laborum qui
        illum amet placeat ea nisi
        sapiente natus corporis quia voluptatem maiores!
      </p>
      <p>
        Lorem ipsum dolor sit amet consectetur adipisicing elit.
        Nesciunt dolor eius rem voluptates aliquid, itaque labore quae laborum qui
        illum amet placeat ea nisi
        sapiente natus corporis quia voluptatem maiores!
      </p>
    </div>
  </div>
);

export default About;
