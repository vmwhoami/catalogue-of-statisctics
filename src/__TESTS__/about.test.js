import React from 'react';
import renderer from 'react-test-renderer';
import About from '../components/About';

it('renders Footer correctly', () => {
  const tree = renderer.create(<About />);
  expect(tree).toMatchSnapshot();
});
