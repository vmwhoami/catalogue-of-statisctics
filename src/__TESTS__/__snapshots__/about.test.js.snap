// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders Footer correctly 1`] = `
<div
  className="container"
>
  <svg
    className="logo"
    id="icon-vmlogo"
    viewBox="0 0 51 32"
  >
    <path
      d="M15.726 6.299l5.030-6.269 30.659 0.023-25.653 31.875-25.652-31.875h8.976l16.676 20.722 10.978-13.64 1.762-2.118h-1.796l-10.944 13.527-4.46-5.554 6.461-7.974-2.018-0.001-5.56 6.838z"
    />
  </svg>
  <div
    className="content"
  >
    <h2>
      THIS  WEBSITE   DISPLAYS  CRYPTO THAT IS  WORTH MORE THAN ONE DOLLAR
    </h2>
    <p>
      Lorem ipsum dolor sit amet consectetur adipisicing elit. Nesciunt dolor eius rem voluptates aliquid, itaque labore quae laborum qui illum amet placeat ea nisi sapiente natus corporis quia voluptatem maiores!
    </p>
    <p>
      Lorem ipsum dolor sit amet consectetur adipisicing elit. Nesciunt dolor eius rem voluptates aliquid, itaque labore quae laborum qui illum amet placeat ea nisi sapiente natus corporis quia voluptatem maiores!
    </p>
  </div>
</div>
`;
